ItsJonQ/g2
# Add AlignmentMatrixControl

### Testing instructions

Ensure the [storybook example for G2](https://g2-git-add-alignment-matrix-control.itsjonq.vercel.app/iframe.html?id=components-alignmentmatrixcontrol--default) matches the [Gutenberg storybook example](https://wordpress.github.io/gutenberg/iframe.html?id=components-alignmentmatrixcontrol--default&viewMode=story).

Things that do not work:

* Arrow key navigation
* Tabbing in and out of the component
* IDs are randomly generated and break the snapshot tests every time they run as a new one is generated each time
